### PR TITLE
add biginteger ut

### PR DESCRIPTION
**Pull request by cloud8little**

### CODE REVIEW

**Review by Tommo-L — APPROVED** [review]

**Comment by shargon** [comment]
Could you apply the changes inside https://github.com/neo-project/neo-devpack-dotnet/pull/433 ?

File: tests/Neo.SmartContract.Framework.UnitTests/HelperTest.cs [hunk 1]
```diff
@@ -66,6 +66,28 @@ public void TestToBigInteger()
...
+            var result = _engine.ExecuteTestCaseStandard("testBigIntegerCast", new VM.Types.ByteString(new byte[] { 0x00, 0x00, 0x8d, 0x49, 0xfd, 0x1a, 0x07 }));
+            Assert.AreEqual(VMState.HALT, _engine.State);
+            Assert.AreEqual(1, result.Count);
+
+            var item = result.Pop();
+            Assert.IsInstanceOfType(item, typeof(Integer));
+            Assert.AreEqual(2000000000000000, item.GetInteger());
+
+            _engine.Reset();
+            result = _engine.ExecuteTestCaseStandard("testBigIntegerParseHexString", new VM.Types.ByteString(new byte[] { 0x00, 0x00, 0x8d, 0x49, 0xfd, 0x1a, 0x07 }));
```
Comment (shargon) [inline tests/Neo.SmartContract.Framework.UnitTests/HelperTest.cs:82]: it doesn't seems a hex string

Comment (Tommo-L) [inline tests/Neo.SmartContract.Framework.UnitTests/HelperTest.cs:82]:
```suggestion
            result = _engine.ExecuteTestCaseStandard("testBigIntegerParseString", (VM.Types.ByteString)"2000000000000000");
```

File: tests/Neo.SmartContract.Framework.UnitTests/TestClasses/Contract_Helper.cs [hunk 2]
```diff
@@ -24,6 +24,16 @@ public static BigInteger TestToBigInteger(byte[] data)
             return data.ToBigInteger();
         }
 
+        public static BigInteger TestBigIntegerCast(byte[] data)
+        {
+            return (BigInteger)(ByteString)data;
+        }
+
+        public static BigInteger TestBigIntegerParseHexString(byte[] data)
```
Comment (Tommo-L) [inline tests/Neo.SmartContract.Framework.UnitTests/TestClasses/Contract_Helper.cs:32]:
```suggestion
        public static BigInteger TestBigIntegerParseString(byte[] data)
```

**Review by Tommo-L — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Pull request merged**